client/my-sites/people/invite-people/index.jsx

```diff
@@ -286,113 +293,175 @@ const InvitePeople = React.createClass( {
 	},
 
 	renderRoleExplanation() {
+		const { translate } = this.props;
 		return (
 			<a
 				target="_blank"
 				rel="noopener noreferrer"
 				href="http://en.support.wordpress.com/user-roles/"
 				onClick={ this.onClickRoleExplanation }
 			>
-				{ this.translate( 'Learn more about roles' ) }
+				{ translate( 'Learn more about roles' ) }
 			</a>
 		);
 	},
 
+	renderInviteForm() {
+		const {
+			site,
+			translate,
+			needsVerification,
+			isJetpack,
+			isSSOActive
+		} = this.props;
+		const inviteForm = (
+			<Card>
+				<EmailVerificationGate>
+					<form onSubmit={ this.submitForm } >
+						<div role="group" className="invite-people__token-field-wrapper">
+							<FormLabel>{ translate( 'Usernames or Emails' ) }</FormLabel>
+							<TokenField
+								isBorderless
+								tokenizeOnSpace
+								autoCapitalize="none"
+								autoComplete="off"
+								maxLength={ 10 }
+								value={ this.getTokensWithStatus() }
+								onChange={ this.onTokensChange }
+								onFocus={ this.onFocusTokenField }
+								disabled={ this.state.sendingInvites } />
+							<FormSettingExplanation>
+								{ translate(
+									'Invite up to 10 email addresses and/or WordPress.com usernames. ' +
+									'Those needing a username will be sent instructions on how to create one.'
+								) }
+							</FormSettingExplanation>
+						</div>
+
+						<RoleSelect
+							id="role"
+							name="role"
+							includeFollower
+							siteId={ this.props.siteId }
+							onChange={ this.onRoleChange }
+							onFocus={ this.onFocusRoleSelect }
+							value={ this.state.role }
+							disabled={ this.state.sendingInvites }
+							explanation={ this.renderRoleExplanation() }
+							/>
+
+						<FormFieldset>
+							<FormLabel htmlFor="message">{ translate( 'Custom Message' ) }</FormLabel>
+							<CountedTextarea
+								name="message"
+								id="message"
+								showRemainingCharacters
+								maxLength={ 500 }
+								acceptableLength={ 500 }
+								onChange={ this.onMessageChange }
+								onFocus={ this.onFocusCustomMessage }
+								value={ this.state.message }
+								disabled={ this.state.sendingInvites } />
+							<FormSettingExplanation>
+								{ translate(
+									'(Optional) You can enter a custom message of up to 500 characters ' +
+									'that will be included in the invitation to the user(s).'
+								) }
+							</FormSettingExplanation>
+						</FormFieldset>
+
+						<FormButton disabled={ this.isSubmitDisabled() } onClick={ this.onClickSendInvites } >
+							{ translate(
+								'Send Invitation',
```

Comment:
@rralian @ebinnion yeah, I'll look into it. https://github.com/Automattic/gp-localci/issues/24